README.md

```diff
@@ -25,7 +25,7 @@ If you prefer to test your manual buildscript before opening the pull request, w
 
 The development version of BinaryBuilder makes use of the new [`Artifacts` system](https://julialang.github.io/Pkg.jl/dev/artifacts/) shipping in Julia 1.3.  This means that BinaryBuilder no longer generates `build.jl` files that are placed into your Julia package's `deps/` folder, but instead generates whole Julia packages (known colloquially as "jll" packages) that are placed within the [JuliaBinaryWrappers organization](https://github.com/JuliaBinaryWrappers/).  Merged pull requests to Yggdrasil result in new versions of these wrapper packages being generated, uploaded and registered, allowing your client Julia code to simply invoke `using LibFoo_jll` to get ahold of your binaries with no need for a `Pkg.build()` step.  (This will, of course, only be the case for Julia 1.3+)
 
-JLL packages created by Yggdrasil are hosted under the [JuliaBinaryWrappers](https://github.com/JuliaBinaryWrappers) GitHub organization.
+JLL packages created by Yggdrasil are hosted under the [JuliaBinaryWrappers](https://github.com/JuliaBinaryWrappers) GitHub organization and are automatically registered in the Julia package registry.  To add an dependency on a JLL package like [Cairo_jll](https://github.com/JuliaBinaryWrappers/Cairo_jll.jl) to your package, add a line `Cairo_jll = "83423d85-b0ee-5818-9007-b63ccbeb887a"` to the `[deps]` (dependencies) section of your `Project.toml` file.  The UUID of a JLL package (e.g. `83423d85-b0ee-5818-9007-b63ccbeb887a`) can be found by looking in the `Project.toml` of the JLL package under [JuliaBinaryWrappers](https://github.com/JuliaBinaryWrappers), for example in [`JuliaBinaryWrappers/Cairo_jll.jl/Project.toml`](https://github.com/JuliaBinaryWrappers/Cairo_jll.jl/blob/master/Project.toml) for Cairo_jll.
```

Comment:
That's how you *manually* add `Cairo_jll`.  I'm talking about how to add a JLL as a dependency of another package so that the JLL is automatically installed.

In practice, this seems to be how JLL packages are almost always used — hardly anyone adds them manually, because they are added as dependencies of the packages that need them.

Reply:
You can instantiate the project of the package and add it in that way.  This is what I usually do to not having to look up the UUID.

Reply:
I don't understand what you guys are getting at.  When projects convert to using JLL files, they add the JLL package as a dependency to their `Project.toml` files.  To do this someone needs to look up the UUID, no?

Reply:
That's correct, but you can also:
```julia
]activate ~/.julia/dev/Cairo
add Cairo_jll
```
which I assume is what Fredrik was suggesting, too

Reply:
Silly me, I've been writing `Project.toml` files by hand all this time, and I didn't even know that `activate` would do this!